docs/user/README.md

```diff
@@ -252,6 +252,18 @@ By using Telegraf, data points can be efficiently streamed into `modelardbd` fro
 [collection of data sources](https://www.influxdata.com/time-series-platform/telegraf/telegraf-input-plugin/)
 such as [MQTT](https://mqtt.org/) and [OPC-UA](https://opcfoundation.org/about/opc-technologies/opc-ua/).
 
+## ModelarDB configuration
+`ModelarDB` can be configured before the server is started using environment variables. A full list of the environment 
+variables is provided here. If an environment variable is not set, the specified default value will be used.
+
+| **Variable**                                     | **Default** | **Description**                                                                                                                                                                                                                                                                                                                                            |
+|--------------------------------------------------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MODELARDBD_PORT                                  | 9999        | The port of the Apache Arrow Flight Server.                                                                                                                                                                                                                                                                                                                |
```

Comment:
If Apache Arrow Flight endpoint is in #144 maybe replace "Server" with "Endpoint" vice versa.

Reply:
I am not sure what you mean. The server has endpoints but the port is related to the server. I have left it as it was.

Reply:
In #144 I suggested calling it an Apache Arrow Flight endpoint (like `AWS_ENDPOINT`) to differentiate it from `modelardbd` which we have been calling the server in the [documentation](https://github.com/ModelarData/ModelarDB-RS/blob/main/docs/user/README.md?plain=1#L29) and in the [source code](https://github.com/ModelarData/ModelarDB-RS/blob/main/crates/modelardb_common/src/types.rs#L105). If you think it is clear enough if we use server for both `modelardbd` and Apache Arrow Flight that is also fine for me. Thus, I have kept this thread unresolved but approved the PR. Feel free to resolve the thread after making a decision on what term to use for Apache Arrow Flight.